docker-compose.yml

```diff
@@ -0,0 +1,28 @@
+# The first instance will automatically bootstrap the cluster.
+# Scale this tier and each additional instance will automatically
+# self-configure as a member of the cluster
+couchbase:
+    image: autopilotpattern/couchbase:latest # enterprise-4.0.0-r1
```

Comment:
Is the `r1` a tag for *our* release of this? i.e. is the image we want to push here `autopilotpattern/couchbase:enterprise-4.0.0-r2`? (and then `r3` for the version using telemetry)